src/Menu.cpp

```diff
@@ -94,7 +94,7 @@ static MenuDef menuDefFile[] = {
     { _TRN("&Open...\tCtrl+O"),             CmdOpen,                   MF_REQ_DISK_ACCESS },
     { "Open Folder",                        CmdOpenFolder,             MF_REQ_DISK_ACCESS | MF_RAMICRO_ONLY },
     { _TRN("&Close\tCtrl+W"),               CmdClose,                  MF_REQ_DISK_ACCESS },
-    { _TRN("&Close All Files"),             CmdCloseAll,               MF_REQ_DISK_ACCESS },
+    { _TRN("&Close All Files\tCtrl+Shift+W"),  CmdCloseAll,             MF_REQ_DISK_ACCESS },

```

Comment:
&Close is a conflict so am suggesting
Close All &Tabs

Reply:
It's done 👍